bundles/org.palladiosimulator.dataflow.confidentiality.analysis/src/org/palladiosimulator/dataflow/confidentiality/analysis/characteristics/variable/DataCharacteristicsCalculatorFactory.java

```diff
@@ -0,0 +1,17 @@
+package org.palladiosimulator.dataflow.confidentiality.analysis.characteristics.variable;
+
+import java.util.List;
+
+import org.palladiosimulator.dataflow.confidentiality.analysis.characteristics.CharacteristicValue;
+import org.palladiosimulator.dataflow.confidentiality.analysis.characteristics.DataFlowVariable;
+
+public interface DataCharacteristicsCalculatorFactory {
```

Comment:
Why is a factory required here while there is no factory required to create the node characteristics calculators?